roles/test_operator/README.md

```diff
@@ -0,0 +1,45 @@
+# Test operator
+
+Execute tests via the [test-operator](https://openstack-k8s-operators.github.io/test-operator/).
+
+## Parameters
+
+* `cifmw_test_operator_artifacts_basedir`: (String) Directory where we will have all test_operator related files. Default value: `{{ cifmw_basedir }}/tests/test_operator` which defaults to `~/ci-framework-data/tests/test_operator`
+* `cifmw_test_operator_namespace`: (String) Namespace inside which all the resources are created. Default value: `openstack`
+* `cifmw_test_operator_index`: (String) Full name of container image with index that contains the test_operator. Default value: `quay.io/openstack-k8s-operators/test-operator-index:latest`
+* `cifmw_test_operator_timeout`: (Integer) Timeout in seconds for the execution of the tests. Default value: `3600`
+* `cifmw_test_operator_logs_image`: (String) Image that should be used to collect logs from the pods spawned by the test-operator. Default value: `quay.io/quay/busybox`
```

Comment:
NOTE: even tough it is small image, at this point you can expect our base image used by all openstack 
pod is already cached on all nodes.

Reply:
Do you know please where is the base image stored?

Reply:
Keeping the public one by default is better, it can be overridden by a ci scenario file.